src/control-plane/backend/lambda/api/service/project.ts

```diff
@@ -37,14 +37,28 @@ export class ProjectServ {
     return pipelines[0];
   }
 
-  private getPresetAppDashboard(pipeline: IPipeline, appId: string) {
+  private getPresetAppDashboard(pipeline: IPipeline, appId: string, realtime: boolean = false) {
     const stackDashboards = getReportingDashboardsUrl(
       pipeline.stackDetails ?? pipeline.status?.stackDetails, PipelineStackType.REPORTING, OUTPUT_REPORT_DASHBOARDS_SUFFIX);
     if (stackDashboards.length === 0) {
       return undefined;
     }
     const appDashboard = stackDashboards.find((item: any) => item.appId === appId);
     if (appDashboard) {
+      if (realtime) {
+        const presetDashboard: IDashboard = {
+          id: appDashboard.realtimeDashboardId,
+          name: 'Realtime Dashboard',
```

Comment:
Do we have an i18n mechanism in the backend?

Reply:
The realtime dashboard name is only default value, will not display on the frontend. And the preset dashboard name will display use i18n.